modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -121,6 +122,48 @@ struct GInferBase {
     }
 };
 
+struct InOutInfo
+{
+    std::vector<std::string> in_names;
+    std::vector<std::string> out_names;
+};
+
+class GInferInputs
+{
+public:
+    cv::GMat& operator[](const std::string& name) { return in_blobs[name]; }
```

Comment:
BTW, python wrapper doesn't know how to wrap `std::string` and I'll have to use `cv::String` instead

Reply:
`cv::String` equals to `std::string`